Cargo.toml

```diff
@@ -69,6 +70,7 @@ libgit2-sys = "0.18.2"
 libloading = "0.8.8"
 memchr = "2.7.5"
 miow = "0.6.0"
+objc2 = "0.6.2"
```

Comment:
> This adds a dependency on [the objc2 ecosystem](https://github.com/madsmtm/objc2) (objc2, block2 and objc2-encode), which I maintain. They are fairly widely used, and Mozilla https://github.com/gfx-rs/wgpu/pull/5641, though I acknowledge that this could still be seen as a supply-chain attack. Implementing the execution policy check without these crates is doable, though a lot more code.